packages/stores/types.ts

```diff
@@ -203,13 +212,15 @@ type TCommonStore = {
     platform: string;
     routeBackInApp: (history: Pick<RouteComponentProps, 'history'>, additional_platform_path?: string[]) => void;
     routeTo: (pathname: string) => void;
+    server_time: Moment;
     changeCurrentLanguage: (new_language: string) => void;
     changeSelectedLanguage: (key: string) => void;
     current_language: string;
     is_language_changing: boolean;
 };
 
 type TUiStore = {
+    addToast: () => void;
```

Comment:
@mahdiyeh-deriv The `addToast` method in `UIStore` accepts an argument, We need to add the types correctly here 🙏🏻